File: ios/AsyncStorageSqliteModule.m
#import <React/RCTBridgeModule.h>

@interface RCT_EXTERN_MODULE(AsyncStorageSqlite, NSObject)

RCT_EXTERN_METHOD(getRandomNumber:(RCTPromiseResolveBlock)resolve
                  reject:(RCTPromiseRejectBlock)reject);

@end

File: ios/AsyncStorageSqliteModule.swift
import Foundation

@objc(AsyncStorageSqlite)
class AsyncStorageSqliteModule : NSObject {
    @objc(getRandomNumber:reject:)
    func random(resolve: RCTPromiseResolveBlock, reject: RCTPromiseRejectBlock) {
        resolve(Int.random(in: 1...100))
    }
}
